jenkins/bundleKeytar.sh

```diff
@@ -15,7 +15,8 @@ chmod +x ./jq
 curl -fs https://$githubAuthHeader@api.github.com/repos/atom/node-keytar/releases/tags/v$keytarVersion |
     ./jq -c '.assets[] | select (.name | contains("node"))' |
     ./jq -cr 'select (.browser_download_url) | .browser_download_url' |
-    while read -r bdu; do curl -fsL -o `echo -n $(echo -n $bdu | md5sum | cut -c1-6)'-'$(basename $bdu)` $bdu; done
+    while read -r bdu; do curl -fsLOJ $bdu; done
```

Comment:
Seems like we no longer want to produce a bunch of `<commit-hash>-keytar-os-arch.tgz` files and instead we are getting the names from a header?

Reply:
The hashes were generated from the file URLs - you're correct that they're no longer needed. They were needed to fake out the downloading of prebuilds into NPM cache, but for local prebuilds no weird hashes are required 😁

Reply:
thank you! : )